backend/tests/api/test_main.py

```diff
@@ -297,3 +318,19 @@ def test_create_tray_collect(self):
         }
         response = self.app.post("/add_tray_collect", json=payload)
         assert response.status_code == 200
+
+    def test_create_tray_return(self):
+        # Add a vendor first
+        vendor = {
+            "name": "hitata",
+            "address": "defeat road",
+            "city": "zootopia",
+        }
+        response = self.app.post("/add_vendor", json=vendor)
+
+        payload = {
+            "return_time": "2020-10-08 11:11:51.291273",
+            "vendor_id": 1,
```

Comment:
Instead of hard coding vendor_id, I think it's better to use id from the response.

Reply:
done!